tools/filter_metadata/README.md

```diff
@@ -0,0 +1,7 @@
+# filter_metadata.py
+This script collects the sizeBytes of bitstreams based on input metadata 
+and the UUID of bitstreams whose sizeBytes are 0.
```

Comment:
I do not properly understand this description. Based on which metadata, which UUID? Maybe add there some example.